pkg/sshd/server.go

```diff
@@ -215,6 +223,19 @@ func (s *server) answer(channel ssh.Channel, requests <-chan *ssh.Request, sshCo
 					req.Reply(ok, nil)
 					break
 				}
+
+				repoName := parts[1]
```

Comment:
@arschles that was already present here https://github.com/deis/builder/pull/170#discussion-diff-53344595R221